indexer-app/src/cmr/indexer/data/concepts/collection/keyword.clj

```diff
@@ -82,9 +82,14 @@
         instruments (concat platform-instruments (mapcat :composed-of platform-instruments))
         instrument-short-names (distinct (keep :short-name instruments))
         instrument-techiques (keep :technique instruments)
-        characteristics (mapcat :characteristics platforms)
-        char-names (keep :name characteristics)
-        char-descs (keep :description characteristics)
+        plat-characteristics (mapcat :characteristics platforms)
+        plat-char-names (keep :name plat-characteristics)
+        plat-char-descs (keep :description plat-characteristics)
+        plat-char-values (keep :value plat-characteristics)
+        instr-characteristics (mapcat :characteristics instruments)
+        instr-char-names (keep :name instr-characteristics)
+        instr-char-descs (keep :description instr-characteristics)
+        instr-char-values (keep :value instr-characteristics)  
```

Comment:
The ticket description is a bit confusing to me. It seems to assume name and description are already indexed, only value needs to be added to the index. Since I didn't see them being indexed, I just indexed them all.
Here is the description:
"Index Platform and Instrument Characteristic value so that it can be searched through the keyword search parameter. Currently Characteristics names and descriptions are indexed as are Additional Attributes names, data types, values, and descriptions but Characteristics values are not indexed. The airborne group from two different DAACs would like to search on the Tail Number of an aircraft platform. But since the value is not indexed, the tail number can't be searched. The name of the Characteristic doesn't help because the name is Tail Number and all aircraft tail numbers are going to have this name."